packages/ui-extensions/docs/surfaces/checkout/reference/examples/ui-components/progress-determinate-state.example.tsx

```diff
@@ -7,20 +7,29 @@ import {
   TextBlock,
 } from '@shopify/ui-extensions-react/checkout';
 
-export default reactExtension('purchase.checkout.footer.render-after', () => (
-  <Extension />
-));
+export default reactExtension(
```

Comment:
Just for extra safety: same question here @oleksandroliynyk ?

Reply:
My assumption here is that we can include this, but I will defer to the author here.

Reply:
Finally, since we don't have confirmation from the author, I will end up removing this for now. This is safer and can be added in upcoming release.